android/library/src/main/java/com/liferay/mobile/screens/ddm/form/model/FormContext.kt

```diff
@@ -0,0 +1,84 @@
+package com.liferay.mobile.screens.ddm.form.model
+
+import com.liferay.apio.consumer.model.Thing
+import com.liferay.apio.consumer.model.get
+
+/**
+ * @author Paulo Cruz
+ */
+class FormContext @JvmOverloads constructor(
+	val isReadOnly: Boolean = false,
+	val isShowRequiredFieldsWarning: Boolean = false,
+	val isShowSubmitButton: Boolean = false,
```

Comment:
Can you apply the Javi suggestion here too? :)

Reply:
Ok, I renamed on every place that had isShow. Thanks!